database-postgres/README.md

```diff
@@ -0,0 +1,123 @@
+
+# Instant OpenHIE Postgres database - docker-swarm
```

Comment:
We should add these docs to the documentation folder for this package so it turns up in gitbook.